apps/sms/test/marionette/lib/messages.js

```diff
@@ -63,11 +64,11 @@ var ParticipantsAccessor = require('./views/participants/accessors');
 
         Participants: new ParticipantsAccessor(client),
 
-        launch: function() {
+        launch() {
           client.switchToFrame();
           client.apps.launch(ORIGIN_URL);
           client.apps.switchToApp(ORIGIN_URL);
-          client.helper.waitForElement(SELECTORS.main);
+          client.helper.waitForElement(SELECTORS.appReady);
```

Comment:
Well, we'll need it in the upcoming patch that revives Inbox tests, see https://github.com/azasypkin/gaia/commit/7aa2b76b0fbbbda38b4809c69ecb652080f30e26#diff-a3f0ad18f54e1750f7f7ecbbc2356f25R71